.gitmodules

```diff
@@ -2,3 +2,7 @@
 	path = third_party/cutlass
 	url = https://github.com/NVIDIA/cutlass.git
 	branch = v2.4.0
+[submodule "third_party/fio"]
+	path = third_party/fio
+	url = https://github.com/axboe/fio.git
+	branch = master
```

Comment:
no need?

Reply:
I think we should be explicit here so that we won't fail silently in the future. What do you think @cp5555 ?

Reply:
It's OK for me.